Algorithms/Sequences/FibonacciSequence.cs

```diff
@@ -19,22 +19,31 @@ public class FibonacciSequence : ISequence
         /// <summary>
         /// Gets Fibonacci sequence.
         /// </summary>
-        public IEnumerable<BigInteger> Sequence
```

Comment:
Please rollback to the previous implementation because it is way more efficient.